### ISSUE TITLE
Add Support for Bitbucket

### ISSUE THREAD

**Issue by followgodstudio** [dataform-co/dataform#1218]
Hi Folks,
Our team want to ask for Bitbucket support. I only saw Github and GitLab are supported.
We can also contribute to the program if needed. I believe that quite some companies are using Bitbucket lately.
Currently we are using BigQuery as our primary storage.

We really appreciate it if this can happen sooner.
Thank you all

**Comment by furlongce** [comment]
Agreed, this would be great to have

**Comment by aportosa** [comment]
Hello there! Any news on this feature? Do you plan to implement a support for Bitbucket?

**Comment by Rhythmfish** [comment]
@aportosa
Have you tried linking Bitbucket yet? 
Even though it's not on the supported list I successfully linked our Dataform repo to our Azure Devops repo. The issue I was able to circumvent was that Dataform does not like %20 (space) characters in the url. I had to create a new Azure project without spaces and was able to link that to Dataform. This might something to investigate with Bitbucket.
